doc/pages/Manual.md

```diff
@@ -1331,12 +1331,14 @@ for <span class="opam">opam</span>.
   specify wrappers around every command executed during the corresponding action
   of any package. The command-line elements will be prefixed to the package
   command, so for example command `[ make "install" ]` with wrapper
-  `[ "time" "-o" "/tmp/timings" "-a" ]` will result in the command
-  `[ "time" "-o" "/tmp/timings" "-a" make "install" ]`.
-  As `init-scripts:` are stored in the hook directory, when using a wrapper
-  script defined by `init-scripts:`, use the variable `%{hook}%` as prefix for
-  you script filename.
-  The filters are evaluated in the same scope as the package commands.
+  `[ "time" "-o" "/tmp/timings" "-a" ]` will result in the command `[ "time"
+  "-o" "/tmp/timings" "-a" make "install" ]`. The filters are evaluated in the
```

Comment:
I am not sure if it's still the case, but at some point a carriage return within the backquotes would break formatting.

Reply:
It doesn't break locally, but a check on server is needed.